### ISSUE TITLE
Render data object and collection description

### ISSUE THREAD

**Issue by noheton** [noheton/shepard#343]
***In GitLab by @xit-paul-reinhardt on Nov 29, 2024, 09:19 GMT+1:***

As a researcher, I want descriptions of my data objects and collections to be rendered like in the editor, so that I am not surprised after saving descriptions.

ACs:

- the description of data objects and collections is displayed with rendered html instead of displaying html tags as text

Hints:

- a view of tiptap edited content was already implemented in #208

**Assignees:** xit-paul-reinhardt

*Migrated from GitLab: https://gitlab.com/dlr-shepard/shepard/-/work_items/343*

**Comment by noheton** [comment]
***In GitLab by @xit-paul-reinhardt on Nov 29, 2024, 09:21 GMT+1:***

marked this issue as blocked by #208

**Comment by noheton** [comment]
***In GitLab by @wiglef on Dec 3, 2024, 09:30 GMT+1:***

added dlr-shepard#8 as parent epic

**Comment by noheton** [comment]
***In GitLab by @xit-paul-reinhardt on Dec 13, 2024, 10:37 GMT+1:***

mentioned in commit ec873bda131b191ef9fd89570a0e92cd50a292b4

**Comment by noheton** [comment]
***In GitLab by @xit-paul-reinhardt on Dec 13, 2024, 11:14 GMT+1:***

Testing Hint: Right now the description is rendered like a lab journal entry. In the UI drafts the font size would be smaller for descriptions. In order to properly style it, I would need to know how to style "legacy" descriptions. That includes the following text styles:

undefined

When editing the descriptions in #320 we can then think about which styles we want to enable for the descriptions, I guess that's out of scope for today.

**Comment by noheton** [comment]
***In GitLab by @xit-paul-reinhardt on Dec 17, 2024, 11:08 GMT+1:***

As aligned on with @KlaraForner and @jessicafriedline we leave it as is as we are actually satisfied with it. Further testing still needed.

**Comment by noheton** [comment]
***In GitLab by @KlaraForner on Dec 17, 2024, 16:29 GMT+1:***

Tested: thumbs up
